examples/with-react-native-web/apps/web/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "web",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "clean": "rm -rf node_modules && rm -rf .next"
```

Comment:
Yarn will hoist all of these node_modules. Interesting to see so many frameworks deeply distrusting of node_modules.

Reply:
Removed